app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def email_link(type = :contact, text: nil, additional_info: nil)
   end
 
   def community_members_label
-    SiteConfig.community_member_label.pluralize
+    Settings::Community.member_label.pluralize
```

Comment:
The DataUpdateScript will be run around 10 min after the deployment, I'm assuming in the meantime before it runs `Settings::Community.member_label` will return the default defined below "user" so besides showing slightly wrong data for a short while this shouldn't break due to a nil value. Do I have that correct?

Reply:
I swear, one day I will remember that data update scripts get scheduled in the future and don't run during the migration stage (for fairly obvious reasons given how long some of them take).

Otherwise, yes, you have that correct. But I'm now wondering if I should move the migration and DUS out into a separate PR that we merge first and rebase this one. That said, we didn't do this for any of the previous models.

Reply:
I actually wanted to make use of my funky time zone and just sneakily deploy in the middle of the night for the US and Europe but I'm currently experiencing deployment issues (see #sre) so I guess that's a no-go.